server/plugin.go

```diff
@@ -513,6 +535,25 @@ func (p *Plugin) onActivate() error {
 			return dbErr
 		}
 
+		// simulate a ping to the plugin from server so it appears "online" immediately.
+		// !!!! remove this when fixed in server
+		_, err = db.Exec("update remoteclusters set lastpingat=$1 where remoteid=$2;", model.GetMillis(), p.remoteID)
+		if err != nil {
+			return fmt.Errorf("cannot simulate ping: %w", err)
+		}
+
+		// writing the invite directly in the DB
+		// !!!! remove this when fixed in server
+		p.inviteRemoteToChannel = func(channelID, remoteID, userID string) error {
```

Comment:
Should this just be a method on the `plugin` instead of a possibly `nil` function pointer (depending on when we initialize)?

Reply:
This is no longer needed.  Handled by https://github.com/mattermost/mattermost/pull/26753